deployment/docker/Makefile

```diff
@@ -7,6 +7,11 @@ CERTS_IMAGE := alpine:3.14.0
 GO_BUILDER_IMAGE := golang:1.16.5
 BUILDER_IMAGE := local/builder:2.0.0-$(shell echo $(GO_BUILDER_IMAGE) | tr : _)
 BASE_IMAGE := local/base:1.1.3-$(shell echo $(ROOT_IMAGE) | tr : _)-$(shell echo $(CERTS_IMAGE) | tr : _)
+VMUI_VERSION=v0.1.0
+
+vmui:
+	cd app/vmselect/ui && \
+	wget  https://github.com/VictoriaMetrics/vmui/releases/download/${VMUI_VERSION}/static.zip -O /tmp/static.zip; unzip /tmp/static.zip -d /tmp; rm -rf ./*; mv /tmp/build/* ./
```

Comment:
nit: extra `space` between `wget` and address.
Also, long command lines can be broke into multiple lines via `\` char.

Reply:
yeah, thanks you take the time to review the PR, fixed, pushed again.